drivers/sensor/adi/adxl345/adxl345.h

```diff
@@ -19,47 +20,143 @@
 #include <zephyr/sys/util.h>
 
 /* ADXL345 communication commands */
-#define ADXL345_WRITE_CMD          0x00
-#define ADXL345_READ_CMD           0x80
-#define ADXL345_MULTIBYTE_FLAG     0x40
+#define ADXL345_WRITE_CMD      0x00
+#define ADXL345_READ_CMD       0x80
+#define ADXL345_MULTIBYTE_FLAG 0x40
 
 /* Registers */
-#define ADXL345_DEVICE_ID_REG      0x00
-#define ADXL345_RATE_REG           0x2c
-#define ADXL345_POWER_CTL_REG      0x2d
-#define ADXL345_DATA_FORMAT_REG    0x31
-#define ADXL345_X_AXIS_DATA_0_REG  0x32
-#define ADXL345_FIFO_CTL_REG       0x38
-#define ADXL345_FIFO_STATUS_REG    0x39
-
-#define ADXL345_PART_ID            0xe5
-
-#define ADXL345_RANGE_2G           0x0
-#define ADXL345_RANGE_4G           0x1
-#define ADXL345_RANGE_8G           0x2
-#define ADXL345_RANGE_16G          0x3
-#define ADXL345_RATE_25HZ          0x8
-#define ADXL345_ENABLE_MEASURE_BIT (1 << 3)
-#define ADXL345_FIFO_STREAM_MODE   (1 << 7)
-#define ADXL345_FIFO_COUNT_MASK    0x3f
-#define ADXL345_COMPLEMENT         0xfc00
-
-#define ADXL345_MAX_FIFO_SIZE      32
+/* ADXL345 DEVICE_ID register */
+#define ADXL345_DEVICE_ID_REG   0x00
+#define ADXL345_DEVICE_ID_VALUE 0b11100101
```

Comment:
This seems like a non-related change; it already was defined in:
``` c
#define ADXL345_PART_ID            0xe5
```

Moreover, Usage of binary constants is discouraged. See: #60564.
Please discard non-related changes.